javascripts/application.js

```diff
@@ -93,7 +93,7 @@ $(document).ready(function() {
       if ($table.length > 0){
         $table.after(inlineNavigation);
       } else {
-        $("main p").first().after(inlineNavigation);
+        $("main > p").first().after(inlineNavigation);
```

Comment:
Mostly just curious what `"main > p"` does? A link to the relevant jQuery documentation is more than sufficient 😄

Reply:
https://developer.mozilla.org/en-US/docs/Web/CSS/Child_combinator